src/gtk/textctrl.cpp

```diff
@@ -124,6 +124,41 @@ static void wxGtkTextApplyTagsFromAttr(GtkWidget *text,
         }
     }
 
+    if( attr.HasFontUnderlinedWithEffect() )
+    {
+        GtkTextTag *tag1;
+        PangoUnderline pg_style;
```

Comment:
Call it `pangoUnderlineStyle` or something else conforming to wx naming convention.

Reply:
Renamed